packages/flame/example/lib/main.dart

```diff
@@ -74,6 +67,13 @@ class Square extends RectangleComponent with TapCallbacks {
     );
   }
 
+  @override
+  void update(double dt) {
+    super.update(dt);
+    angle += speed * dt;
+    angle %= 2 * math.pi;
```

Comment:
tau?

Reply:
I was thinking that too, but maybe not on the most basic example so that we don't confuse newcomers?